crates/bevy_window/src/event.rs

```diff
@@ -4,7 +4,8 @@ use super::{WindowDescriptor, WindowId};
 use bevy_math::{IVec2, Vec2};
 
 /// A window event that is sent whenever a window's logical size has changed.
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, PartialEq)]
```

Comment:
Is it possible to derive `Copy` for the items in this file?

Reply:
A number of them have `String` or `PathBuf` fields, so `Copy` cannot be derived for those.